### DISCUSSION TITLE
Using PopForums v17.0.0 with projects using Framework

### DISCUSSION THREAD

**Discussion by jdgunaseelan** [POPWorldMedia/POPForums#236] [Q&A] [answered]
I am trying to use some of the methods in a class that uses .NET Framework. I am unable to reference PopForums because they target different frameworks. Is there a way to use them or would I have to use the older version of PopForums that uses .NET Framework?

**Comment by jeffputz** [comment, marked as answer]
You'll have to go back to the old days of v13.x to use the old .NET Framework.

**Reply by jdgunaseelan** [reply]
That's what I expected, thanks!
